docs/reference/api/storage/storage.md

```diff
@@ -27,13 +27,15 @@ The `fopen` function is similar to the open function above but associates a stre
 
   - **Wear leveling** - Because the most common form of embedded storage is erodible flash memories, this file system provides a form of dynamic wear leveling for systems that cannot fit a full flash translation layer.
 
-- **FATFileSystem** - The FAT file system is a well-known file system that you can find on almost every system, including PCs. The Mbed OS implementation of the FAT file system is based on ChanFS and is optimized for small embedded systems.
+- **FATFileSystem** - The FAT file system is an established disk-oriented file system that you can find on most operating systems, including Windows, Linux, Mac OS X and Mbed OS.
 
```

Comment:
We need a link to page with details about FATFileSystem to make it consistent compared to LittleFS.

Reply:
Ah good catch, I believe this should link to the FATFileSystem.md also in this PR.

@AnotherButler, are you able to add this link so it will work on the doc site?